widget_driver/example/lib/widgets/coffee_community/coffee_library/coffee_detail/coffee_detail_page_driver.dart

```diff
@@ -5,20 +5,20 @@ import '../../../../models/coffee.dart';
 part 'coffee_detail_page_driver.g.dart';
 
 @GenerateTestDriver()
-class CoffeeDetailPageDriver extends WidgetDriver {
-  final int index;
-  final Coffee _coffee;
+class CoffeeDetailPageDriver extends WidgetDriver with $CoffeeDetailPageDriverProvidedPropertiesMixin {
```

Comment:
One thought: It gets pretty long this name.. of the mixin.. `$CoffeeDetailPageDriverProvidedPropertiesMixin`
Should be just skip some parts here.. and make it shorter? 
We could just name it:
`_$DriverProvidedPropertiesMixin`

This has 2 benefits:
1: The name is shorter.. So you don't get this bloated definition of your driver.
2: It is always the same name of the mixin which you add to your driver.. So it gets a bit more uniform.. Easier to remember.. Everytime I add a providableProperty, then I also just add `_$DriverProvidedPropertiesMixin` at the end of my class definition.. What do you think?